extmod/vfs_blockdev.c

```diff
@@ -78,8 +78,6 @@ void mp_vfs_blockdev_init(mp_vfs_blockdev_t *self, mp_obj_t bdev) {
 
 int mp_vfs_blockdev_read(mp_vfs_blockdev_t *self, size_t block_num, size_t num_blocks, uint8_t *buf) {
     if (self->flags & MP_BLOCKDEV_FLAG_NATIVE) {
-        // CIRCUITPY-CHANGE: Pass the blockdev object into native readblocks so
-        // it has the corresponding state.
```

Comment:
this still appears to be a change compared to upstream.

Reply:
Added back. I misunderstood what upstream did.